config/routes.rb

```diff
@@ -7,6 +7,10 @@
 
   get "/signup", to: "users#new"
   resources :users, only: %i(new create show)
+
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
```

Comment:
giả sử a dùng method get cho logout được không e ?

Reply:
Nếu cùng đổi delete thành get ở 2 nơi là:
- <%= link_to t(".logout"), logout_path, method: :delete %>
- delete "/logout", to: "sessions#destroy"
Thì em nghĩ là code vẫn chạy và nó vẫn sẽ logout cho mình nhưng mà như thế thì ko đúng với logic đặt ra trong restAPI, get chỉ nên dùng với cái action có trả về view.
Em nghĩ như thế ạ